includes/api.inc

```diff
@@ -38,14 +38,20 @@ function islandora_bookmark_get_user_owned_bookmarks($type = NULL) {
 
   // User is anon, use session as well.
   if ($user->uid == 0) {
-    if (!isset($_SESSION) || empty($_SESSION['islandora_bookmark']) && variable_get('islandora_bookmark_create_user_default_lists', FALSE)) {
-      BookmarkDefaultSessionList::createNewList(t('My Default List'), 'bookmark_default', $user);
+    if (!isset($_SESSION) || !isset($_SESSION['islandora_bookmark']) || empty($_SESSION['islandora_bookmark']) && variable_get('islandora_bookmark_create_user_default_lists', FALSE)) {
+      $bookmark_list[] = BookmarkDefaultSessionList::createNewList(t('My Default List'), $user, 'Default bookmark');
     }
 
     if (!empty($_SESSION['islandora_bookmark'])) {
       foreach (array_keys($_SESSION['islandora_bookmark']) as $id) {
         $bookmark_list[] = BookmarkSession::getList($id);
       }
+      if (count($_SESSION['islandora_bookmark']) == 1) {
```

Comment:
I need to look at that because I can´t remember really the exact" why". I slightly remember thinking anonymous users, which are a "bookmarking" burden, should not trust or rely on having multiple lists around and having them to be kept safely around because it depends on cookies) and we don´t even notify them that this storage is just session based and volatile.  But again, need to dig deeper to know the why? Will have a solution/answer tomorrow. Thanks!